### ISSUE TITLE
broken license pages on spdx.org

### ISSUE THREAD

**Issue by ungoldman** [spdx/license-list-XML#615]
Hello! I was asked by pombreda in #spdx on IRC to post about this issue here.

I've noticed at least two license pages on https://spdx.org are broken:

https://spdx.org/licenses/Apache-2.0

![screen shot 2018-02-18 at 5 23 00 pm](https://user-images.githubusercontent.com/427322/36359461-6c8bb0d8-14d0-11e8-9d19-736c552d63be.png)

https://spdx.org/licenses/ISC

![screen shot 2018-02-18 at 5 23 09 pm](https://user-images.githubusercontent.com/427322/36359462-6d97019e-14d0-11e8-944c-6595b473738c.png)

A friend pointed out that adding `.html` to the URL seems to fix the rendering issue. Can these pages be fixed, or should I just no longer link to them and use the `.html` version instead?

**Comment by goneall** [comment]
This looks like a configuration problem with our webserver.  @kestewart - can you forward this on to the appropriate Linux Foundation person?

As a work-around, you can use the html file extensions (e.g. ISC.html) and it will render correctly.

**Comment by waldyrious** [comment]
Those errors indicate that the document is malformed xml. This means the pages are being served as xhtml. I don't think working around the errors (to make sure the documents are parsed with a more lenient process) is the best choice here; why not simply fixing the actual errors? This is a simple matter of self-closing the `br` tags, i.e. changing `<br>` to `<br/>`.

**Comment by goneall** [comment]
@waldyrious I agree.  In looking back at the source there are some <br> tags that are not closed.  @kestewart - you can ignore my previous request.  

I need to track down the source of the problem.

**Comment by goneall** [comment]
Found the problem in the licenseGenerator application.  Working on a fix.

**Comment by goneall** [comment]
Fix the problem in the LicenseListPublish tool and pushed updated files to spdx.org/licenses website.  @jlovejoy just FYI - I kept the original index and licenses.json file but replaced all the HTML files with regenerated HTML to fix this problem.  I used the license-list-XML with the v3.0 tag.  I don't recall if there were any updates to the website after 3.0.  If so, they would need to be replaced.

Let me know if you see any problems.
